src/blocks/cpu.rs

```diff
@@ -71,13 +73,15 @@ impl ConfigBlock for Cpu {
         Ok(Cpu {
             id,
             update_interval: block_config.interval,
-            output: TextWidget::new(id, 0, shared_config).with_icon("cpu")?,
+            output: TextWidget::new(id, 0, shared_config.clone()).with_icon("cpu")?,
```

Comment:
I think you can avoid using `clone()` by moving this line under `boost_icon_off`.

Reply:
Cool! You are right. I thought, fields must be initialized in the declaration order.